lambda-natchez/shared/src/main/scala/feral/lambda/natchez/package.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda
+
+import _root_.natchez.Kernel
+import feral.lambda.events.ApiGatewayProxyEvent
+import feral.lambda.events.ApiGatewayProxyEventV2
+import feral.lambda.events.DynamoDbStreamEvent
+import feral.lambda.events.KinesisStreamEvent
+import feral.lambda.events.S3BatchEvent
+import feral.lambda.events.SqsRecordAttributes
+import feral.lambda.natchez.KernelSource
+import org.typelevel.ci._
+
+protected trait KernelSources {
```

Comment:
Instead of putting these in the package object, we can put them in the companion object for `KernelSource`. Then they will automatically be in implicit scope without an import.

Reply:
@alexcardell Can you look into this as well? 🙏

Reply:
Will do

Reply:
@armanbilge done :)